docs/source/markdown/podman-network-connect.1.md

```diff
@@ -0,0 +1,35 @@
+% podman-network-connect(1)
+
+## NAME
+podman\-network\-connect - Connect a container to a network
+
+## SYNOPSIS
+**podman network connect** [*options*] network container
+
+## DESCRIPTION
+Connects a container to a network. You can connect a container by name or by ID. Once connected, the container can communicate with other containers
+in the same network.
+
+To verify what networks a given container is connected to, use the `podman network inspect` command. Use `podman network disconnect` to remove a container from the network.
```

Comment:
`podman network inspect` shows nothing about what networks a container is connected to. `podman container inspect` on a container would do that.